drafts/19008-security.adoc

```diff
@@ -8,76 +8,103 @@ seo-title: Securing Open Liberty apps and microservices - OpenLiberty.io
+Open Liberty also supplies the `samlWeb-2.0` feature for authenticating with a SAML security provider, and the `spnego-1.0` feature
+for authenticating with Microsoft Active Directory. 
 
-Developers can also build and consume their own JWTs using an API with the JWT (`jwt-1.0`) feature. This enables them to build JWTs for use with microservices without using an OpenID Connect provider.
+== Consuming Json Web Tokens (JWT)
+Developers can use the MicroProfile JWT (`mpjwt-1.1`) feature to secure microservices.  JWTs are identity credentials with a standardized format that can be exchanged between processes. They're secured against tampering by use of digital signatures.  These tokens can be submitted to secured microservices to propagate the identity of a logged-in user in an efficient, self-contained way.  
 
+Developers can write apps that consume JWTs from an external provider, or they can build and consume their own JWTs using an API with the JWT (`jwt-1.0`) feature. This enables them to build JWTs for use with microservices. 
 
 == Configuring Open Liberty as a provider
-It is also possible to configure Open Liberty as a provider for some or all of these services. The OpenID Connect Server (`openidConnectServer-1.0`) feature can be configured to manage user authentication and to issue JWTs for use by OpenID Connect clients and microservices. The OpenID Connect Server can work directly with a user registry such as LDAP or other security provider, or to configure a database.
-
-Developers can also configure applications so that, after the user has logged in once, they do not have to log in again to every application separately. Opne Liberty provides a SAML client (`samlWeb-2.0`) for use with a SAML identity provider and a Spnego client (`spnego-1.0`) for use with Microsoft domain controllers.
-
+It is also possible to configure Open Liberty as an OpenID Connect provider. The OpenID Connect Server (`openidConnectServer-1.0`) feature can be configured to manage user authentication and to issue JWTs for use by OpenID Connect clients and microservices. The OpenID Connect Server can work directly with a user registry such as LDAP or act as an identity broker and delegate to another security provider.
 
 //
 
 == Example: Using OpenID Connect and JWTs to authenticate users and secure microservices:
 
-The following sample code and configurations use three servers to demonstrate authenticating a user and accessing a secured microservice using a JWT. The three pieces of this are:
+The following sample code and configurations use two servers to demonstrate authenticating a user, building a JWT and accessing a secured microservice using that JWT. The pieces of this are:
```

Comment:
Based on the next couple of comments below...does this work?
"The following sample code and configurations show two web services to demonstrate authenticating a user, building a JWT, and accessing a secured microservice using that JWT. Each web service is hosted on its own Open Liberty server:"